source/draw/gpu/opengl/YuvOptimizedContext.ooc

```diff
@@ -25,7 +25,7 @@ YuvOptimizedContext: class extends OpenGLContext {
 	_eglBin := RecycleBin<EGLYuv> new(100, |image| image _recyclable = false; image free())
 	init: func (other: This = null) {
 		super(other)
-		Debug error(!queryExtension("GL_EXT_YUV_target"), "vidhance_render_mode_t::VIDHANCE_RENDER_MODE_OPENGL_YUV_OPTIMIZED is not supported on this device")
+		Debug error(!queryExtension("GL_EXT_YUV_target"), "The chosen render mode is not supported on this device")
```

Comment:
either this or `RENDER_MODE_OPENGL_YUV_OPTIMIZED` would be fine too, you decide.

Reply:
I like that more, 1 sec